### PR TITLE
[Blog] ServiceMeshCon 2022 talk with Intel - MeshMark

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>
**Description**
ServiceMeshCon 2022 talk with Intel - MeshMark

This PR fixes #

**Notes for Reviewers**
MERGE AFTER 17 MAY Central Time only
This blog will boost SEO :)

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 51be01b10bc501cb6e060bcbb7afcbe21629b036 at: https://62801d7f523926396bb6d5ac--layer5ng.netlify.app

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    So like this you can measure and create other MUEs. We will look at how you can visualize this within an environment and so let's look at the demonstration. So let's jump into a sibling CNCF project called Meshery.
+    Meshery is a cloud native management plane. Users of Meshery can configure their Kubernetes deployments any and every service mesh as well as on board and off board their workloads onto any given mesh.
+</p>    
+
+<img src={MeshMapDemo} alt="MeshMap demo" style="width:40%;float:right;"  />
+<h3>Lee demonstrates MeshMap with an example of console application</h3>
+
+<p>
+Let's take an example workload a console application, load it into the visual designer, take a look at the service splitting functionality of console and note in this case we're assigning a weight of three when we can change that to four to derive its mesh Mark which is a mesh utilization efficiency calculation of the efficiency by which that network function is being performed.
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:122]:
```suggestion
Let's take an example workload a  Consul application, load it into the visual designer, take a look at the service splitting functionality of console and note in this case we're assigning a weight of three when we can change that to four to derive its mesh Mark which is a mesh utilization efficiency calculation of the efficiency by which that network function is being performed.
```

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    <img src={MUE} alt="MeshMark MUE" style="width:50%;float:left;padding-right:1rem;" /> And you would want to see if the CPU performance as a ratio to the available resources is a loss or a gain. So CPU performance, raw loss over total CPU is our MUE one.
+    And that's just one minus CPU utilization over 100. That's a very simple ratio and if you see on the slide, the graph shows you that as the latency increases, your Mue lowers. And so that's a very good indicator that your efficiency of your infrastructure is not very good because your latencies are increasing as your QPS increases.
+    So like this you can measure and create other MUEs. We will look at how you can visualize this within an environment and so let's look at the demonstration. So let's jump into a sibling CNCF project called Meshery.
+    Meshery is a cloud native management plane. Users of Meshery can configure their Kubernetes deployments any and every service mesh as well as on board and off board their workloads onto any given mesh.
+</p>    
+
+<img src={MeshMapDemo} alt="MeshMap demo" style="width:40%;float:right;"  />
+<h3>Lee demonstrates MeshMap with an example of console application</h3>
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:119]:
```suggestion
<h3>Lee demonstrates MeshMap with an example Consul application</h3>
```

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 3c86b46202ce74bcb2718f848cb461c8696305a5 at: https://628076904adca66839e72f38--layer5ng.netlify.app

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 3]
```diff
@@ -0,0 +1,130 @@
+---
+title: "MeshMark: Service Mesh value measurement"
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:2]:
```suggestion
title: "MeshMark: Cloud Native Value Measurement"
```

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 4]
```diff
@@ -0,0 +1,130 @@
+---
+title: "MeshMark: Cloud Native Value Measurement"
+date: 2022-05-17 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./banner.png
+description: "ServiceMeshCon 2022 talk with Intel"
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:6]:
```suggestion
description: "An open index for measuring performance of cloud native infrastructure in context of the value provided to your business."
```

**Comment by l5io** [comment]
🚀 Preview for commit 5f8fa362ceb7f976bc89f3ddee0d110f9899812d at: https://62809b0552fefc7c219e5917--layer5ng.netlify.app

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 5]
```diff
@@ -0,0 +1,130 @@
+---
+title: "MeshMark: Cloud Native Value Measurement"
+date: 2022-05-17 10:30:05 -0530
+author: Gaurav Chadha
+thumbnail: ./banner.png
+description: "An open index for measuring performance of cloud native infrastructure in context of the value provided to your business."
+type: Blog
+category: "MeshMark"
```
Comment (leecalcote) [inline src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:8]:
```suggestion
category: "Service Mesh Performance"
```

File: src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+tags:
+ - MeshMark
+ - ServiceMeshCon
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import Blockquote from "../../../../reusecore/Blockquote";
+import slidesTAG from "../../../events/kubecon-china-2021/cncf-tag-network-and-service-mesh-working-group-kubecon-china-2021-lee-calcote-ken-owens.pdf";
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/collections/blog/2022/2022-05-14-service-meshcon-talk-intel/index.mdx:19]: Thanks

**Comment by leecalcote** [comment]
Make sure that MeshMark is always one word. It shows as two words in a few places.

**Pull request merged**